conda-forge.yml

```diff
@@ -1 +1,3 @@
 conda_forge_output_validation: true
+os_version:
+  linux_64: cos7
```

Comment:
I don't know whether this is the recommended way to specify a Linux image. Commenting here so we can get a check from a conda-forge core dev.

Reply:
The alternative might to use `recipe/conda_build_config.yaml`

Reply:
This is how a core dev did it for pytorch
https://github.com/conda-forge/pytorch-cpu-feedstock/blame/master/conda-forge.yml

Reply:
Thanks!